src/serialMIDI.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include "midi_Namespace.h"
+
+BEGIN_MIDI_NAMESPACE
+
+template <class SerialPort>
+class serialMIDI
+{
+public:
+	serialMIDI(SerialPort& inSerial)
+        : mSerial(inSerial)
+	{
+	};
+
+public:
+	void begin(MIDI_NAMESPACE::Channel inChannel = 1)
+	{
```

Comment:
This should probably call `mSerial.begin` ?

Reply:
The opposite, the MidiInterface will call this method.
See in MIDI.hpp:

```
template<class Transport, class Settings, class Platform>
void MidiInterface<Transport, Settings, Platform>::begin(Channel inChannel)
{
    // Initialise the Transport layer
    mTransport.begin();
```